src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -64,6 +67,9 @@ public ArrayList()
         // initially empty, but will have room for the given number of elements
         // before any reallocations are required.
         // 
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the specified initial capacity.</summary>
```

Comment:
@mairaw correct.  You will need to both tag the nodes that you want to be simplified, in this case the `cref`, and then call the `Simplifier` on the tree.  There is an example in code formatter that you should be able to pattern off of:

https://github.com/dotnet/codeformatter/blob/849fcf60f6b6d85a34e28a9a4d42d82b5ed03116/src/Microsoft.DotNet.CodeFormatting/Rules/ExplicitThisRule.cs

@cartermp correct.  That code needs to pick up the correct set of references for the file.  Alternatively you can use the `MSBuildWorkspace` on the appropriate project file and it will populate with the correct references.